Source/Private/_ASPendingState.mm

```diff
@@ -1095,13 +1095,8 @@ - (void)applyToView:(UIView *)view withSpecialPropertiesHandling:(BOOL)specialPr
     view.clipsToBounds = _flags.clipsToBounds;
 
   if (flags.setBackgroundColor) {
-    // We have to make sure certain nodes get the background color call directly set
-    if (specialPropertiesHandling) {
-      view.backgroundColor = [UIColor colorWithCGColor:backgroundColor];
-    } else {
-      // Set the background color to the layer as in the UIView bridge we use this value as background color
-      layer.backgroundColor = backgroundColor;
-    }
+    view.backgroundColor = [UIColor colorWithCGColor:backgroundColor];
```

Comment:
If we set the background color to the view the layer should automatically be updated. Could you double cbeck that this is not the case.

Reply:
@maicki - We tested this today and surprisingly, the layer is not updated (at least not immediately)

Reply:
Confirm what Rahul said: the view doesn't propagate the background color immediately which causes many of our tests to fail because they use the layer to render snapshots. For example [this test](https://github.com/TextureGroup/Texture/blob/master/Tests/ASDisplayNodeSnapshotTests.mm#L19) failed because the parent node has a `nil` background color by the time the test takes a snapshot using [`ASSnapshotVerifyNode`](https://github.com/TextureGroup/Texture/blob/master/Tests/ASSnapshotTestCase.h#L21).

An alternative approach would be updating `ASSnapshotVerifyNode` to use the view if the node is view-backed. But by doing so, we're accepting the risk that there is a time window in which the view and the layer disagree (which we're already doing). And this time window is undocumented by UIKit.

Given that now UIColor and UIView has a new dynamic capability, and that we should rely on UIView more to take advantage of this new capability, we should update the framework to set to both the view and the layer and thus eliminate the time window and its associated risks.